pkg/executor/join/hash_join_v2.go

```diff
@@ -957,10 +966,7 @@ func (w *ProbeWorkerV2) runJoinWorker() {
 	if w.HashJoinCtx.stats != nil {
 		start := time.Now()
 		defer func() {
-			t := time.Since(start)
```

Comment:
why not let `runJoinWorker` take `start` as its argument as `restoreAndProbe`

Reply:
> why not let `runJoinWorker` take `start` as its argument as `restoreAndProbe`

done